### PR TITLE
Anthony's review :tada: 

### PR DESCRIPTION
**Pull request by abretaud**
Hello!
Finally took the time to review your tuto, it's super cool, well done :clap:!
I've made a few typo corrections/cleanup/suggestions, feel free to accept them or not :)
Other remarks:
- in the geoc-gxit repo, the Dockerfile and Makefile should be capitalized to match with what's done usually
- you say that image should be pushed to biocontainers: I'm not sure it's needed. It's recommended for normal tools, but for gxit I think pushing to docker hub or quay.io is the usual way to do it. (maybe I'm wrong here)
- `Check XML syntax with xmlvalidation.com or w3schools XML validator, or use a linter in your code editor.` -> planemo lint should work (but warn that planemo test will not)
- problem with code formatting in ansible section

### CODE REVIEW

File: topics/dev/tutorials/interactive-tools/tutorial.md [hunk 1]
```diff
@@ -60,11 +60,11 @@ The environment_variables would be needed to retrieve data from Galaxy History.
 
 This tutorial demonstrates how to build and deploy a Galaxy Interactive Tool (GxIT). GxITs are accessible through the Galaxy tool panel, like any installed Galaxy tool. Our example application is a simple R Shiny app that we call `Tabulator`.
 
-There are three elements to a GxIT - an application script, a Docker container and a Galaxy tool XML file. This tutorial will take you through creating those components, and installing them as a new interactive tool into a local Galaxy instance and an existing Galaxy instance.
+There are three elements to a GxIT - an application script, a Docker container and a Galaxy tool XML file. This tutorial will take you through creating those components, and installing them as new interactive tools into a local Galaxy instance and an existing Galaxy instance.
```
Comment (neoformit) [inline topics/dev/tutorials/interactive-tools/tutorial.md]: I would keep the original grammar as we do use multiple components to install a single IT

**Review by neoformit — CHANGES_REQUESTED** [review]
Thanks Anthony! One teeny change, everything else looks great.

**Comment by neoformit** [comment]
Thanks for the revisions! Biocontainers - we (Galaxy AU) had a discussion about image hosting and thought it would be better to push remotes to a central repository rather than personal Docker Hubs etc. Should make for better production stability into the future. Happy to hear any thoughts on that.

File: topics/dev/tutorials/interactive-tools/tutorial.md (hunk 1 above)
Comment (abretaud) [reply topics/dev/tutorials/interactive-tools/tutorial.md]: Ok you're right!

Comment (abretaud) [reply topics/dev/tutorials/interactive-tools/tutorial.md]:
```suggestion
There are three elements to a GxIT - an application script, a Docker container and a Galaxy tool XML file. This tutorial will take you through creating those components, and installing them as a new interactive tool into a local Galaxy instance and an existing Galaxy instance.
```

**Comment by abretaud** [comment]
> Thanks for the revisions! Biocontainers - we (Galaxy AU) had a discussion about image hosting and thought it would be better to push remotes to a central repository rather than personal Docker Hubs etc. Should make for better production stability into the future. Happy to hear any thoughts on that.

Yeah ok, biocontainers is a good way to ensure long term availability, but maybe add a note saying something like "we encourage to you to use biocontainers, but if your image is on hub.docker.com or quay.io it is fine, as long as you link it to a public github repo and make sure someone will maintain it in the long term (bus factor > 1)"?

**Comment by neoformit** [comment]
> Yeah ok, biocontainers is a good way to ensure long term availability, but maybe add a note saying something like "we encourage to you to use biocontainers, but if your image is on hub.docker.com or quay.io it is fine, as long as you link it to a public github repo and make sure someone will maintain it in the long term (bus factor > 1)"?

Ok that sounds like a compromise - will update it to read like this. Thanks Anthony.

**Review by neoformit — APPROVED** [review]

**Pull request merged**